File: app/src/main/java/com/pru/pinmm/model/payloads/LoginPayload.java
package com.pru.pinmm.model.payloads;


import com.google.gson.annotations.SerializedName;

public class LoginPayload {
    @SerializedName("emailId")
    String emailId;
    @SerializedName("password")
    String password;
    @SerializedName("loginMenuId")
    String loginMenuId;

    public String getEmailId() {
        return emailId;
    }

    public void setEmailId(String emailId) {
        this.emailId = emailId;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getLoginMenuId() {
        return loginMenuId;
    }

    public void setLoginMenuId(String loginMenuId) {
        this.loginMenuId = loginMenuId;
    }
}

File: app/src/main/java/com/pru/pinmm/model/payloads/LoginPayload.kt
package com.pru.pinmm.model.payloads

import com.google.gson.annotations.SerializedName

data class LoginPayload(
    @SerializedName("emailId")
    var emailId: String? = null,
    @SerializedName("password")
    var password: String? = null,
    @SerializedName("loginMenuId")
    var loginMenuId: String? = null
)